pkgs/development/haskell-modules/configuration-ghc-9.10.x.nix

```diff
@@ -120,4 +123,8 @@ self: super: {
   lukko_0_1_2 = dontCheck super.lukko_0_1_2; # doesn't compile with tasty ==1.4.*
   resolv = dontCheck super.resolv; # doesn't compile with filepath ==1.5.*
   primitive-unlifted = dontCheck super.primitive-unlifted; # doesn't compile with primitive ==0.9.*
+
+  haskell-language-server = disableCabalFlag "retrie" (disableCabalFlag "hlint" (disableCabalFlag "stylishhaskel" (super.haskell-language-server.override {stylish-haskell = null;retrie = null;apply-refact=null;hlint = null;})));
```

Comment:
Do you want to convert this to use `lib.pipe` as discussed on matrix?

Reply:
i think 3 is about the limit where doing directly still is acceptable.

Reply:
Well, I personally use lib.pipe as soon as I hit 2 functions, but sure.